caffe2/onnx/onnx_exporter.cc

```diff
@@ -337,6 +350,58 @@ ConvertedResult OnnxExporter::CommonCaffe2OpToOnnxNodes(
   return result;
 }
 
+ConvertedResult OnnxExporter::CreateBinaryElementwiseOpNodes(
+    const caffe2::OperatorDef& def,
+    const std::unordered_map<std::string, caffe2::TensorShape>& shapes) {
+  // The upper bound (excluded) of expanded y.
+  // 0 indicates that no need to unsqueeze y, otherwise, we have to do the unsqueeze.
+  caffe2::OperatorDef mdef(def);  // The modified def without broadcast and axis
+  const auto& x = mdef.input(0);
+  const auto& y = def.input(1);  // Refer to the old def, later won't change it.
+  const auto& z = mdef.output(0);
+  const auto& x_shape = shapes.at(x);
+  const auto& y_shape = shapes.at(y);
+  for (int i = 0; i < mdef.arg_size(); ++i) {
+    const auto& arg = mdef.arg(i);
+    if (arg.name() == "broadcast") {
+      if (i < mdef.arg_size() - 1) {
+        mdef.mutable_arg()->SwapElements(i, mdef.arg_size() - 1);
+      }
+      mdef.mutable_arg()->RemoveLast();
+      break;
+    }
+  }
+  std::vector<int64_t> axes;
+  for (int i = 0; i < mdef.arg_size(); ++i) {
+    const auto& arg = mdef.arg(i);
+    if (arg.name() == "axis") {
+      int64_t axis = arg.i();
+      if (x_shape.dims().size() - axis != y_shape.dims().size()) {
+        int64_t end_dim = y_shape.dims().size() - 1 - axis + x_shape.dims().size();
+        for (int64_t j = y_shape.dims().size(); j < end_dim; ++j) {
+          axes.emplace_back(j);
```

Comment:
`std::vector::resize(size_type count, const value_type& value )`

Reply:
I guess `std::iota` also works.

Reply:
oh yeah it should be `iota`, `resize` is wrong.

Reply:
it has to be `resize` + `iota`